airtime_mvc/application/models/Schedule.php

```diff
@@ -269,8 +269,11 @@ public static function getPreviousCurrentNextMedia($utcNow, $currentShowInstance
                 $previousFile = CcFilesQuery::create()
                     ->filterByDbId($previousMediaFileId)
                     ->findOne();
-                $previousMediaName = $previousFile->getDbArtistName() . " - " . $previousFile->getDbTrackTitle();
-                $previousMetadata = CcFiles::sanitizeResponse($previousFile);
+                if (isset($previousFile))
+                {
```

Comment:
Slight style point - please put opening brackets on the same line as the if statement